ansible/roles_ocp_workloads/ocp4_workload_shared_cluster_access/tasks/post_workload.yml

```diff
@@ -1,9 +1,32 @@
 ---
 # Implement your Post Workload deployment tasks here
-# --------------------------------------------------
+#--- --------------------------------------------------
+- name: Get console route
+  command: oc get route -n openshift-console console -o json
```

Comment:
can we use k8s_info module instead of dropping down into shell or command?

Reply:
Changed to k8s_info